resolwe_bio/tools/check_bam_source.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3
+"""Check if the source of RNA ae rRNA or globin RNA."""
+
+import argparse
+import os
+import sys
+
+MAPPINGS_DIR = '/opt/chrom_mappings/globin_rrna/'
+QC_FILES = {
+    'Homo sapiens': [
+        'Homo_sapiens_rRNA.txt',
+        'Homo_sapiens_globin_RNA.txt',
+    ],
+    'Mus musculus': [
+        'Mus_musculus_rRNA.txt',
+        'Mus_musculus_globin_RNA.txt'
+    ],
+    'Rattus norvegicus': [
+        'Rattus_norvegicus_rRNA.txt',
+        'Rattus_norvegicus_globin_RNA.txt'
+    ],
+}
+
+
+def parse_arguments():
+    """Parse command line arguments."""
+    parser = argparse.ArgumentParser(description=__doc__)
+    parser.add_argument('--chromosome_names', help="File with chromosome names")
+    parser.add_argument('--species', help="Species")
+    return parser.parse_args()
+
+
+def parse_file(filename):
+    """Parse qc files."""
+    qc_file = set()
+    with open(filename) as handle:
+        for line in handle:
+            line = line.strip().split()
+            qc_file.add(line[0])
+    return qc_file
+
+
+def parse_qc(species):
+    """Parse file with chromosome names for globin RNA and rRNA."""
+    qc_file = set()
+    for basename in QC_FILES[species]:
+        filename = os.path.join(MAPPINGS_DIR, basename)
+        qc_file.update(parse_file(filename))
+    return qc_file
+
+
+def check_rna_source(chrom_names, qc_files):
+    """
+    Check if the source of RNA ae rRNA or globin RNA.
+
+    If chromosome names are subset of rRNA or globin RNA chromosome names,
+    exit with status 0 since you don't need bigwig for this file.
+    """
+    chrom_names = parse_file(chrom_names)
+    if chrom_names.issubset(qc_files):
```

Comment:
Rename variables:
`if chromosomes.issubset(globin_rrna_sequences):`
`...`
This make's it much clearer what is actually happening and makes docstring almost redundant.